README.md

```diff
@@ -13,6 +17,6 @@ This repository is licensed under CC0 1.0 Universell (CC0 1.0) Public Domain Ded
 * [protégé](https://protege.stanford.edu/)
 
 ## Contributing and developping
-Please read the [wiki](https://github.com/OpenEnergyPlatform/ontology/wiki) for information about the ontology and its standards.
+Please read the [wiki](https://github.com/OpenEnergyPlatform/ontology/wiki) for information about the ontology, its standards and best practise principles on content and BFO classification.
```

Comment:
Maybe rewrite a bit to:
Please read the [wiki](https://github.com/OpenEnergyPlatform/ontology/wiki) for information about the ontology, its standards, its best practice principles and on BFO classification

Reply:
@han-f , can I suggest you make the comments as a suggestion directly in the [File changes tab](https://github.com/OpenEnergyPlatform/ontology/pull/122/files)? You can access it from the "File changes" tab below this PR's title at the top of the page. The procedure is described [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request).

In my view this process has three advantages : 
 1. It is faster: if the person agrees with your suggestion they can commit it directly from github

 2. It is easier not to forget a suggestion done that way

 2. It binds the git handles of the two people in a commit, making it more transparent on who's behind a commit (i.e. it gives the person commenting also some credits)